metadata/latest/Azure.AI.Vision.Common.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Name": "Azure.AI.Vision.Common",
+  "Version": "0.11.1-beta.1",
```

Comment:
This file should go in the `metadata/preview/` folder because it has a preview version.